### PR TITLE
Add English verbs query file

### PR DESCRIPTION
**Pull request by andrewtavis**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [x] I have updated the [CHANGELOG](https://github.com/scribe-org/Scribe-Data/blob/main/CHANGELOG.md) with a description of my changes for the upcoming release (if necessary) 

---

### Description

This PR includes the query for all English verbs. There are a few points here:
- The English verbs model is quite lean as the goal was to not have "unneeded data"
  - We thus don't have forms for all conjugations, but only those that are different
  - In the formatting process we still need to map them to all of them to make sure that the app references don't need to be rewritten for this case
- "To be" needs to get hard coded within the verbs formatting process as the data structure is dramatically different and adding in all of the conditional selections was getting really annoying
  - Let's discuss if we need to check other irregular verbs
- Checked by talking about this a bit at CCCamp with one of the main Wikidata Lexeme devs/community members 😊

- No directly related issue 🙃 Tangentially related to #39

### CODE REVIEW

**Comment by andrewtavis** [comment]
Look at that CI roll! @SaurabhJamadagni, please review and @wkyoshida can confirm 😊

**Comment by andrewtavis** [comment]
Also, would be great if one of you could do the [maintainer checklist](https://github.com/scribe-org/Scribe-Data/pull/40#issuecomment-1684537220) 😇🙃

**Comment by SaurabhJamadagni** [comment]
> Also, would be great if one of you could do the https://github.com/scribe-org/Scribe-Data/pull/40#issuecomment-1684537220 😇🙃

This basically means checking off the todos on the checklist that was generated with this PR right? Will do along with the review :)

**Comment by andrewtavis** [comment]
> This basically means checking off the todos on the checklist that was generated with this PR right? Will do along with the review :)

That it does! Thank you, @SaurabhJamadagni 🙏

**Review by SaurabhJamadagni — APPROVED** [review]
Thanks for the comments separating out the sections @andrewtavis. Really helped with the review. Looks great to me! Cool thing ruling out contractions too. That hadn't occurred to me. I hope you had fun with the optionals haha 😄

**Comment by SaurabhJamadagni** [comment]
How to check off the maintainer checklist? I don't see an edit option nor will it keep the checkbox saved when I refresh the page.

**Comment by andrewtavis** [comment]
I got the checkboxes for you, @SaurabhJamadagni :) Looks like you two need more rights if it’s not working 🙃

Btw, check Matrix 😉😊

File: src/scribe_data/extract_transform/English/verbs/query_verbs.sparql [hunk 1]
```diff
@@ -1,28 +1,53 @@
...
-    ?simplePastForm ontolex:representation ?simplePast ;
-      wikibase:grammaticalFeature wd:Q1392475 ;
+    ?lexeme ontolex:lexicalForm ?simpPastForm .
+    ?simpPastForm ontolex:representation ?simpPast ;
+    wikibase:grammaticalFeature wd:Q1392475 ;
+    FILTER NOT EXISTS { ?presFPSForm wdt:P6191 wd:Q181970 . }
+    FILTER NOT EXISTS { ?presFPSForm wikibase:grammaticalFeature wd:Q126473 . }
   } .
 
   # Past Participle
```
Comment (wkyoshida) [inline src/scribe_data/extract_transform/English/verbs/query_verbs.sparql:41]: Going off of what we'd do with the `past participle`, i.e. when formatting, to simply use the `past participle` to construct the different conjugations for `* perfect` tenses.. 

I can't remember if we discussed the same for the `present participle` to construct the different conjugations for `* perfect continuous`, e.g. `have been walking`. Is that something that we do/will do? Do we need to grab the `present participle` as well?

Comment (andrewtavis) [reply src/scribe_data/extract_transform/English/verbs/query_verbs.sparql:41]: We haven't been doing continuous tenses until now, which is a good thought that maybe we do need this 🤔 I'd argue against present continuous, but maybe there's a use for perfect continuous :) I'm going to merge so we can play around with this a bit more, but have added a note for the weekly 😊

**Pull request merged**